tests/integ/test_dataframe.py

```diff
@@ -2207,26 +2272,17 @@ def test_fillna(session, local_testing_mode):
         [Row(1, 1.1), Row(None, 1.1)],
     )
 
-    if not local_testing_mode:  # TODO: enable this after rebasing on support-variant
-        df = session.create_dataframe(
-            [[[1, 2], (1, 3)], [None, None]], schema=["col1", "col2"]
-        )
-        Utils.check_answer(
```

Comment:
Where did this example go?

Reply:
This is duplicate code I forgot to remove, this test case is present in `test_replace_with_coercion`.